apps/desktop/dmg/settings.py

```
@@ -0,0 +1,36 @@
+
+app_path = Path(defines["app"]).resolve()  # noqa: F821
+background_path = Path(defines["background"]).resolve()  # noqa: F821
+volume_icon_path = Path(defines["volume_icon"]).resolve()  # noqa: F821
+
+app_name = app_path.name
+files = [str(app_path)]
+symlinks = {"Applications": "/Applications"}
+icon = str(volume_icon_path)
+background = str(background_path)
+
+format = "UDZO"
```

Ruff (0.16.2)

[error] Variable `format` is shadowing a Python builtin

(A001)